### PR TITLE
Add userID parameter in all request handlers

### PR DESCRIPTION
**Pull request by zkokelj**
### Why this change is needed

Please provide a description and a link to the underlying ticket

### What changes were made as part of this PR

Please provide a high level list of the changes made

### PR checks pre-merging

Please indicate below by ticking the checkbox that you have read and performed the required
[PR checks](https://github.com/obscuronet/obscuro-internal/blob/main/dev-ops-docs/dev-pr-checks.md)

- [ ] PR checks reviewed and performed

### CODE REVIEW

File: tools/walletextension/userconn/user_conn.go [hunk 1]
```diff
@@ -154,6 +166,15 @@ func (w *userConnWS) IsClosed() bool {
 	return w.isClosed
 }
 
+func (w *userConnWS) ReadRequestParams() map[string]string {
+	params := make(map[string]string)
+	queryParams := w.req.URL.Query()
+	for key, value := range queryParams {
+		params[key] = value[0]
+	}
+	return params
```
Comment (tudor-malene) [inline tools/walletextension/userconn/user_conn.go:175]: these 2 functions seem identical.
They could both use a util function that takes a query and returns a map

**Review by tudor-malene — COMMENTED** [review]
minor comment

File: tools/walletextension/userconn/user_conn.go (hunk 1 above)
Comment (zkokelj) [reply tools/walletextension/userconn/user_conn.go]: fixed.

**Review by tudor-malene — APPROVED** [review]
lgtm

**Pull request merged**